src/poliastro/util.py

```diff
@@ -69,11 +70,17 @@ def transform(vector, angle, axis='z', unit=None):
 
 def norm(vec):
     """Norm of a Quantity vector that respects units.
-
     """
     return np.sqrt(vec.dot(vec))
 
 
+@jit
+def norm_3d(vec):
```

Comment:
Let's be more consistent with how we `@jit` methods here:

```
def circular_velocity(k, a):
    ...

circular_velocity_fast = jit(circular_velocity)

def norm(vec):
    ...

norm_fast = jit(norm)
```

Notice though that this will require `numba>=0.39`. Also, you might have to do

```
@jit
def norm_fast(vec):
    return np.sqrt((1.0 * vec).dot(1.0 * vec))
```

Reply:
So whenever we jit a function we should use `norm_fast` inside, and whenever we want to retain astropy units we should use `norm`. There's no alternative.